internal/processor/filterset/strict/options.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strict
+
+// SFSOption is the type for strict filtering options that can be passed to NewStrictFilterSet.
+type SFSOption func(*strictFilterSet)
```

Comment:
Same here, just `Option` is enough. However, I don't see any options implemented. What's the purpose of defining this type?

Reply:
Renamed both to just Option.

The main purpose of defining the Option even though there are none is to make the boilerplate code (constructor, eventual yaml config, eventual Factory) between filtersets as consistent as possible and to make it less disruptive in the future if an option does need to be added. 

For example, both the NewRegexpFilterSet & NewStrictFilterSet are exported and accept Options as their last arg to avoid running into the situation in the future where we need something like a New*FilterSet() &  New*FilterSetWithOptions for each filterset implementation.